src/app/dev/DevToys.Tools/Tools/Converters/Date/DateConverterGuiTool.cs

```diff
@@ -0,0 +1,1022 @@
+                            Stack()
+                                .Vertical()
+                                .SmallSpacing()
+                                .WithChildren(
+                                    _dstTicksLabel
+                                )
+                        ),
+                        Cell(
+                            DateConvertDstGridRow.DateTime,
+                            DateConvertDstGridColumn.LeftTitle,
+                            Stack()
+                                .Vertical()
+                                .SmallSpacing()
+                                .WithChildren(
+                                    Label()
+                                        .Style(UILabelStyle.Body)
+                                        .Text("Local DateTime")
+                                )
+                        ),
+                        Cell(
+                            DateConvertDstGridRow.DateTime,
+                            DateConvertDstGridColumn.LeftContent,
+                            Stack()
+                                .Vertical()
+                                .SmallSpacing()
+                                .WithChildren(
+                                    _dstLocalDateTimeLabel
+                                )
+                        ),
+                        Cell(
+                            DateConvertDstGridRow.DateTime,
+                            DateConvertDstGridColumn.RightTitle,
+                            Stack()
+                                .Vertical()
+                                .SmallSpacing()
+                                .WithChildren(
+                                    Label()
+                                        .Style(UILabelStyle.Body)
+                                        .Text("UTC DateTime")
+                                )
+                        ),
+                        Cell(
+                            DateConvertDstGridRow.DateTime,
+                            DateConvertDstGridColumn.RightContent,
+                            Stack()
+                                .Vertical()
+                                .SmallSpacing()
+                                .WithChildren(
+                                    _dstUtcDateTimeLabel
+                                )
+                        )
+                    )
+            )
+            );
+    }
+
+    private IUIDropDownListItem[] BuildTimeZoneItems()
+    {
+        ReadOnlyCollection<TimeZoneInfo> systemTimeZone = TimeZoneInfo.GetSystemTimeZones();
+        string timeZoneSelectedId = _settingsProvider.GetSetting(timeZoneIdSettings);
+        var timeZoneDropDownItems = new IUIDropDownListItem[systemTimeZone.Count];
+
+        if (!Regex.IsMatch(systemTimeZone.ElementAt(0).DisplayName, @"^\(UTC.*\).+$"))
+        {
+            // version < .Net6
+            // This implementation mitigates the changes in the strings
```

Comment:
This is super interesting and a very good catch! How did you find out there was this change between .NET versions?

Reply:
This portion of code is from the version 1

Reply:
I see! :D